contrib/raftexample/main.go

```diff
@@ -33,9 +33,14 @@ func main() {
 	confChangeC := make(chan raftpb.ConfChange)
 	defer close(confChangeC)
 
+	commitC, errorC := make(chan *string), make(chan error)
+	walReady := make(chan struct{})
+	kv := newKVStore(proposeC, commitC, errorC, walReady)
```

Comment:
`newKVStore` blocks until `replayWal` sends `nil` to its `commitC`.
Since we call this before `startRaft`, `newKVStore` either needs to be called inside `startRaft` after `replayWal` or with goroutine like this. Any suggestion? Thanks!

Couldn't figure out an easier way